jdrfandroidbleparser/src/main/java/org/ehealthinnovation/jdrfandroidbleparser/encodedvalue/dis/bgmmeasurement/Type.kt

```diff
@@ -0,0 +1,21 @@
+package org.ehealthinnovation.jdrfandroidbleparser.bgm.encodedvalue.bgmmeasurement
+
+enum class Type constructor(val key: Int) {
+
+    RESERVED_FOR_FUTURE_USE(0),
+    WHOLE_BLOOD_CAPILLARY(1),
```

Comment:
PLease use the naming similar to the Gatt spec. CAPILLARY_WHOLE_BLOOD, CAPILLARY_PLASMA, etc

It makes reading and matching easier.

Reply:
done